tensorflow/core/ops/array_ops.cc

```diff
@@ -1353,8 +1353,8 @@ this operation will permute `params` accordingly.
 out-of-bound indices result in safe but unspecified behavior, which may include
 raising an error.
 
-<div style="width:70%; margin:auto; margin-bottom:10px; margin-top:20px;">
-<img style="width:100%" src="../../../images/Gather.png" alt>
+<div style='width:70%; margin:auto; margin-bottom:10px; margin-top:20px;'>
+<img style='width:100%' src='../../images/Gather.png' alt>
```

Comment:
I'm not sure this works -- can you link to absolute path?

@wolffg @MarkDaoust FYI -- is this part of the docs docs? I don't know what the latest guidance is.